src/installer/installer.go

```diff
@@ -193,23 +200,44 @@ func (i *installer) waitForControlPlane(ctx context.Context, kc k8s_client.K8SCl
 		i.log.Errorf("Timeout waiting for master nodes, %s", err)
 		return err
 	}
+
 	if err := kc.PatchEtcd(); err != nil {
 		i.log.Error(err)
 		return err
 	}
+
+	err := i.addPolicyForController()
+	if err != nil {
+		i.log.Error(err)
+		return err
+	}
+
 	i.waitForBootkube(ctx)
+
 	return nil
 }
 
 func (i *installer) UpdateHostStatus(newStatus string) {
 	i.log.Infof("Updating node installation status: %s", newStatus)
 	if i.HostID != "" {
-		if err := i.ic.UpdateHostStatus(newStatus); err != nil {
+		if err := i.ic.UpdateHostStatus(newStatus, i.HostID); err != nil {
 			i.log.Errorf("Failed to update node installation status, %s", err)
 		}
 	}
 }
 
+func (i *installer) addPolicyForController() error {
```

Comment:
Please move it to the the kubeclient.go.
and make it generic

Reply:
we need it generic? i think we don't. Hope to delete it one day

Reply:
Delete it now if you can.
Why not make it generic?
Isn't it just `oc --config... some command`

Reply:
it doesn't worth to add dependency on ops to client to run it.

Reply:
yap it is. 
Can't delete, still didn't find any other way to do it.
but let it be here. Don't want to add dependency to k8client